interface/usergroup/npi_lookup.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/**
+ * NPI Lookup Backend Proxy
+ *
+ * Proxies requests to NPPES API to avoid CORS issues
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2025 Stephen Waite <stephen.waite@cmsvt.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Csrf\CsrfUtils;
+use GuzzleHttp\Client as Client;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Exception\ConnectException;
+
+// Check authorization
+if (!AclMain::aclCheckCore('admin', 'practice')) {
+    http_response_code(403);
+    echo json_encode(['error' => 'Unauthorized']);
+    exit;
+}
+
+// Verify CSRF token for security
+if (!empty($_GET['csrf_token'])) {
```

Comment:
doing this will mean it bypasses if don't have a csrf_token (ie. no security benefit since can send a $_GET without this variable to bypass the check)
Recommend checking for $_GET instead.

Reply:
side note is that this script has no state changes (ie. no database changes) in it, so does not really need csrf check. However, would still do it in case that gets added in future and no downside to checking it.